components/create-collective/CreateCollectiveForm.js

```diff
@@ -209,6 +219,18 @@ class CreateCollectiveForm extends React.Component {
             <Formik validate={validate} initialValues={initialValues} onSubmit={submit} validateOnChange={true}>
               {formik => {
                 const { values, handleSubmit, errors, touched } = formik;
+                const suggestedSlug = () => {
+                  return values.name
+                    .trim()
+                    .split(' ')
+                    .join('-')
+                    .toLowerCase();
```

Comment:
It's already a good start but we need to have a strategy for international characters (ie: european accents, asian alphabets). In some other places, we're using `slugify` for that.

Reply:
Thanks you for reviewing. I'll check it out and make apt changes. :)